src/Implementation/Renderers/CylinderRenderer.ts

```diff
@@ -0,0 +1,46 @@
+import { IRenderer } from "../../Interfaces/IRenderer";
+import { Workspace } from "@rbxts/services";
+
+const DEFAULT_RADIUS = 0.05;
+
+const recyclables = new Array<CylinderHandleAdornment>();
```

Comment:
This maintains the instance instantiation optimization from the earlier iteration while not making it a forced feature of all projectiles